BuildingBlocks/src/BuildingBlocks.Infrastructure/Persistence/BlobStorage/BlobStorageServiceCollectionExtensions.cs

```diff
@@ -41,7 +41,7 @@ public static void AddBlobStorage(this IServiceCollection services, BlobStorageO
 
 public class BlobStorageOptions
 {
-    public string ConnectionInfo { get; set; } = string.Empty;
-    public string Container { get; set; } = string.Empty;
-    public string CloudProvider { get; set; } = string.Empty;
+    public string ConnectionInfo { get; set; }
+    public string Container { get; set; }
+    public string CloudProvider { get; set; }
```

Comment:
Make them required please.

Reply:
This was left like that on purpose because making them required breaks some of the tests. The problem is in the lines instancing `BlobStorageOptions` such as this one in the `AddBlobStorage` method in the same file:
```csharp
    var options = new BlobStorageOptions();
```

Reply:
Okay. 👍